deploy.bat

```diff
@@ -17,4 +17,7 @@ dotnet build Snowflake.Data\Snowflake.Data.csproj -c Release --force -v n /p:Sig
 
 dotnet pack Snowflake.Data\Snowflake.Data.csproj -c Release --force -v n --no-build  --output %ROOT_DIR%
 
+aws s3 cp s3://sfc-eng-jenkins/repository/net/sign-artifact.exe .
+sign-artifact.exe sign-artifact -o snowflakedb -r snowflake-connector-net -t v%VERSION%  -l 20 -v -u -f Snowflake.Data.%VERSION%.nupkg

```

Comment:
shouldn't we be verifying the integrity of this file in S3? Can we hardcode a hash at a minimum?

Reply:
Added a verification